src/Components/PaperEditor/TranscriptsContainer/TranscriptMenu/index.js

```diff
@@ -8,70 +8,62 @@ import ButtonGroup from 'react-bootstrap/ButtonGroup';
 import DropdownButton from 'react-bootstrap/DropdownButton';
 import LabelsList from '../LabelsList/index.js';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faHighlighter, faCog } from '@fortawesome/free-solid-svg-icons';
+import { faCog, faTag } from '@fortawesome/free-solid-svg-icons';
+import './index.scss';
 
 const TranscriptMenu = (props) => {
-  const labels = props.labels;
+  const { labels, activeLabel } = props;
 
-  const HighlightButton = (
+  useEffect(() => {
+    if (labels) {
+      const defaultLabel = labels.find((label) => label.label === 'Default');
+      if (!activeLabel) props.onLabelSelect(defaultLabel);
+    }
+  }, [ activeLabel, labels, props ]);
+
+  const LabelButton = (
     <Button
       variant="outline-secondary"
       data-label-id={ 'default' }
-      onClick={ (e) => props.handleCreateAnnotation(e) }
+      onClick={ (e) => {
+        props.handleCreateAnnotation(e);
+      } }
     >
-      <FontAwesomeIcon icon={ faHighlighter } flip="horizontal" /> Highlight
+      <FontAwesomeIcon icon={ faTag } flip="horizontal" />
+      <span className='TranscriptMenu__label-button-text'>Label</span>
+      <Col
+        style={ { backgroundColor: activeLabel?.color } }
```

Comment:
Nice little conditional object reference or whatever the heck it's called 😆

Reply:
hahaha thanks! It finally worked, and it felt soooo goooooood 😎